iroh-net/src/magicsock/node_map/node_state.rs

```diff
@@ -1310,6 +1362,24 @@ impl RemoteInfo {
     pub fn has_send_address(&self) -> bool {
         self.relay_url.is_some() || !self.addrs.is_empty()
     }
+
+    /// Merges and deduplicates the [`Source`] and the [`Duration`] since they were discovered for each direct address, returning a list of the most recent [`Source`]s.
```

Comment:
Lots of these comments are full line width, this one alone sits at 168 column width. In the past I have noticed @flub has a 70 or 80 (can't remember) column width setup and I have a 100 one. Both these mean (as far as I remember) a 69/79/99 setup. 

I digress. My point is: we don't have a standard for comment width and enforce one is probably a low priority, high effort, low return endeavor. But even then, most of this comments fall under "extremely long" category for me, and would appreciate having them be 70/80/90/100 width at most

Reply:
authoritative argument for 100 width
https://github.com/rust-lang/rust/tree/HEAD/src/doc/style-guide/src

Reply:
heh.  I must admit I never tweaked how comments are formatted by my editor.  I realise they're shorter than most people, since most people use the 100-char limit.  But I also admit that I secretly often have only space for 92 characters in my editor so I kind of like the shorter comments formatting...

I agree that enforcing a standard on comment width is too much effort unless someone writes it into rustfmt.  I do think the 100-char max line-width should apply though.

As for doc comments, I usually refer to the docs style guide which says the first line should be a short summary.  It doesn't give any limits but again I take that to mean the code formatting rules of a single line, so max 100 characters including indentation.